indico/modules/events/editing/service.py

```diff
@@ -148,3 +150,25 @@ def service_handle_new_editable(editable):
     except requests.RequestException as exc:
         logger.exception('Failed calling listener for editable')
         raise ServiceRequestFailed(exc)
+
+
+def service_handle_review_editable(editable, revision):
```

Comment:
Shouldn't we also send the type of review action that was performed to the service?

Reply:
Maybe, but wouldn't that be also available in the `final_state`? It is set right before going into the service.

Reply:
I think having to check the final state to guess what action was performed isn't particularly developer-friendly ;)